pyramid_oereb/lib/readers/municipality.py

```diff
@@ -38,5 +41,7 @@ def read(self):
             The list of all found records. Since these are not filtered by any criteria the list simply
             contains all records delivered by the source.
         """
+        log.debug("read()")
```

Comment:
Are these kind of log really useful ?

Reply:
Regarding PR #97 : you mean closing the PR 97 and instead I will suggest logging entries in error cases within this PR (302)? Yes, that would be ok for me.

Regarding your question "really useful": this one I guess probably not... except to have a trace (in debug Level) to understand where a request went through. What do you think about the traces with parameter values, would these be (more) useful?